### DISCUSSION TITLE
Structurizr onprem showing empty workspace

### DISCUSSION THREAD

**Discussion by martinkra** [structurizr/onpremises#58] [Q&A] [answered]
### Description

I've started onprem on my machine using docker pull/run. I am able to access the tool on localhost:8080. Also, I can create workspaces. The problem is that it shows that workspace is empty, eventhough I added workspace.dsl into the folder. The file is from getting started:

workspace {

    model {
        user = person "User" "A user of my software system."
        softwareSystem = softwareSystem "Software System" "My software system."

        user -> softwareSystem "Uses"
    }

    views {
        systemContext softwareSystem "SystemContext" {
            include *
            autoLayout
        }

        styles {
            element "Software System" {
                background #1168bd
                color #ffffff
            }
            element "Person" {
                shape person
                background #08427b
                color #ffffff
            }
        }
    }
    
}

### Steps to reproduce

docker pull structurizr/onpremises:latest
docker run...
access localhost:8080 using browser
create new workspace
open workspace page - workspace is empty
access the folder created in previous step
put workspace.dsl into the folder
open workspace page again - workspace is still empty

### Severity

Minor

### Priority

Low

### Resolution

I have no budget, please fix this for free

**Comment by simonbrowndotje** [comment, marked as answer]
The on-premises installation doesn't work in that way, so you can't copy a `workspace.dsl` file into the Structurizr data directory. Your options for creating workspace content are shown when you create a workspace via the UI:

<img width="1245" alt="Screenshot 2023-08-26 at 11 42 56" src="https://github.com/structurizr/onpremises/assets/1009874/beafa157-eae7-4f90-91cd-59d270be8ebd">

If this is not what you were expecting, perhaps you were looking for [Structurizr Lite](https://structurizr.com/help/lite)?

**Reply by martinkra** [reply]
Structurizr Lite works the way I expected. Thx.
